src/coreComponents/linearAlgebra/interfaces/hypre/HypreSolver.cpp

```diff
@@ -340,6 +346,13 @@ void HypreSolver::solve_krylov( HypreMatrix & mat,
     HYPRE_BoomerAMGSetAggNumLevels( uu_amg_solver, 1 );
     HYPRE_BoomerAMGSetNumFunctions( uu_amg_solver, 3 );
 
+    // Relaxation options: Use options 18 or 7.
+    // Coarsening options: Only PMIS is supported
+    // Interpolation options: Use options 3, 6, 14 or 15.
+    HYPRE_BoomerAMGSetRelaxType( uu_amg_solver, 7 );
+    HYPRE_BoomerAMGSetCoarsenType( uu_amg_solver, 8 );
+    HYPRE_BoomerAMGSetInterpType( uu_amg_solver, 3 );
```

Comment:
Options set as specified by @oseikuffuor1.

Reply:
@rrsettgast I don't see where the initialization function (which has hypre_init() ) is called. It should be called prior to calling other hypre functions. The finalize function can be called at the end of the calls to hypre.